### ISSUE TITLE
Laurie Halifax audio

### ISSUE THREAD

**Issue by johnwalley** [CityRC/Equipment#268]
### Boat name or ID

Laurie Halifax

### Problem description

Audio to speakers cuts out intermittently. Does not affect stern-most speaker. When it cuts out it tends to stay out, i.e. it will last the whole outing/race.

**Comment by 13enedict** [comment]
It's probably water ingress; I couldn't find the heat shrink when I fixed the bow speaker. Easily fixed with liquid electrical tape. The centre speaker also has bare wires

**Comment by NSinclair94** [comment]
I will get some heat shrink from work

**Comment by 13enedict** [comment]
You'll need to re-solder it to fit heat shrink...

**Comment by NSinclair94** [comment]
Yeah that's fine, I'll pick up a soldering iron and some solder too

**Comment by bjarr21** [comment]
Yesterday (Sun) managed to have the speaker in 3 seat working, but did not work in bow.  Seemed that not having equipment in the portal at 4 seat helped, maybe it was shifting the wires?  But either way, no sound in bow :(

**Comment by NSinclair94** [comment]
Hopefully, this is sorted.

**Comment by NSinclair94** [comment]
Laurie has four speakers, the Cox box is only designed to support three. This may be  why the back one/two only work intermittently

**Comment by NSinclair94** [comment]
Almost there, bit more to do Monday night

**Comment by 13enedict** [comment]
How go this?

**Comment by bjarr21** [comment]
Still not complete.  The speaker is currently stored in 4's shoe and is
taped down for the outing.  But the speaker doesn't extend to the bow and 2
seat.  Needs to be affixed.
